_rules/SC1-3-1-aria-required-owned-element.md

```diff
@@ -0,0 +1,243 @@
+---
+name: ARIA required owned elements
+
+description: |
+	The rule checks that a role has at least one of its required owned elements.
+
+success_criterion:
+- 1.3.1 # Info and Relationships (A)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required owned elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+For each test target at least one instance of one [required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria) is present as a [semantic role](#semantic-role) for an element [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by the test target.
```

Comment:
I've got the same concern here. IMO this thing is difficult to read. If we break the sentence up, things should be easier to read.

Reply:
The end of the sentence is just a repetition (but somehow upside down 🙃) of the beginning ? isn't it?

Proposal : 

> For each test target, at least one instance of one [required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria), is present.

That is a sufficient expectation ?